k8s-deployer/scripts/publish-report.sh

```diff
@@ -48,6 +53,9 @@ then
   exit 1
 fi
 
+# The caller is expected to set "USERNAME" env variable
+AUTHOR_ARG="${USERNAME} <${USERNAME}@kindredgroup.com>"
```

Comment:
As this is OS repo, and the report can be pushed to any git repo, we shouldn't have `@kindredgroup.com` as suffix to the author.

Reply:
Yes makes sense. I will add a separate ticket to fix it.